packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -151,6 +151,11 @@ export class RemoteMediaGroup {
     throw new Error(`remote media object ${remoteMedia.id} not found in the group`);
   }
 
+  public setPreferLiveVideo(preferLiveVideo: boolean) {
+    this.options.preferLiveVideo = preferLiveVideo;
+    this.sendActiveSpeakerMediaRequest(false);
```

Comment:
hardcoded commit=false means that nothing will happen, the change will not take effect. We need `commit: boolean` to be added as an argument to `setPreferLiveVideo()`.